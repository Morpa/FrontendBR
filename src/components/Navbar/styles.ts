import styled, { css } from 'styled-components'

export const Wrapper = styled.nav`
  ${({ theme }) => css`
    padding: ${theme.spacings.xxsmall};
    cursor: pointer;

    img {
      height: 8rem;
    }
  `}
`
