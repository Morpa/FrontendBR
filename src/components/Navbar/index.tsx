import Link from 'next/link'
import * as S from './styles'

const Navbar = () => (
  <S.Wrapper>
    <Link href="/">
      <img
        src="/img/logo.png"
        alt="Um cÃ­rculo com as cores do Brasil"
        aria-label="frontend-br"
      />
    </Link>
  </S.Wrapper>
)

export default Navbar
