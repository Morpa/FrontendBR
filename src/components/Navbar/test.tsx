import { screen } from '@testing-library/react'
import { renderWithTheme } from 'utils/tests/helpers'

import Navbar from '.'

describe('<Navbar />', () => {
  it('should render the heading', () => {
    const { container } = renderWithTheme(<Navbar />)

    expect(
      screen.getByRole('img', { name: /frontend-br/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
