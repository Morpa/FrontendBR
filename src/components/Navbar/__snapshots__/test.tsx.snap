// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> should render the heading 1`] = `
.c0 {
  padding: 0.8rem;
  cursor: pointer;
}

.c0 img {
  height: 8rem;
}

<nav
  class="c0"
>
  <img
    alt="Um cÃ­rculo com as cores do Brasil"
    aria-label="frontend-br"
    src="/img/logo.png"
  />
</nav>
`;
